client/signup/steps/emails/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'calypso/signup/step-wrapper';
+import EmailTitanCard from 'calypso/components/emails/emails-titan-card';
+import EmailSideExplainer from 'calypso/components/emails/emails-side-explainer';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+
+class EmailsStep extends React.Component {
+	constructor( props ) {
+		super( props );
+	}
+
+	getSideContent = () => {
+		return (
+			<div className="emails__email-side-content-container">
+				<div className="emails__email-side-content">
+					<EmailSideExplainer />
+				</div>
+			</div>
+		);
+	};
+
+	renderContent() {
+		const content = (
+			<div className="register-email-step">
+				<CalypsoShoppingCartProvider>
+					<EmailTitanCard
+						siteUrl={ this.props.progress.domains.siteUrl }
+						addButtonContent={ 'Add' }
+						skipButtonContent={ 'Skip' }
+						onAddButtonClick={ () => {} }
+						onSkipButtonClick={ () => {} }
+						price={ '3.5' }
+					></EmailTitanCard>
+				</CalypsoShoppingCartProvider>
+			</div>
+		);
+		const sideContent = this.getSideContent();
+
+		return (
+			<div
+				key={ this.props.step + this.props.stepSectionName }
+				className="emails__step-content emails__step-content-email-step"
+			>
+				{ content }
+				{ sideContent }
+			</div>
+		);
+	}
+
+	handleSkip = () => {};
+
+	render() {
+		const { flowName, translate, stepName, positionInFlow } = this.props;
+		const backUrl = '/emails/';
```

Comment:
This feels like it shouldn't be hard-coded -- can you add a comment as to why this is here and/or what cleanup needs to happen?

Reply:
Hello! I felt that there were many backUrls to go back. I agree that this should be dynamic, as any other step does. But 
for this being dynamic, we need to know which step/workflow we are to easily go back to. 

I also think that this might be a good improvement for the steps component, it should be selfaware of what's the previous step or where this step is coming from, as it knows where to go for the next step. :)